cyrw/ioctl.go

```diff
@@ -471,9 +470,6 @@ func (d *Device) rxEvent(packet []byte) (err error) {
 		}
 	}()
 	// Split packet into BDC header:payload.
-	if len(packet) < whd.BDC_HEADER_LEN+72 {
```

Comment:
It is redundant with DecodeEvent but there should still be a check so that DecodeBDCHeader does not panic.

Reply:
Then how about this?

```
if len(packet) < whd.BDC_HEADER_LEN {
    return ...
}
bdcHdr = whd.DecodeBDCHeader(packet)
```

Reply:
That's good!